sky/spot/spot_state.py

```diff
@@ -103,6 +103,9 @@ class SpotStatus(enum.Enum):
     # Terminal statuses
     # SUCCEEDED: The job is finished successfully.
     SUCCEEDED = 'SUCCEEDED'
+    # CANCELLING: The job is requested to be cancelled by the user, and the
+    # controller is cleaning up the spot cluster.
+    CANCELLING = 'CANCELLING'
     # CANCELLED: The job is cancelled by the user.
```

Comment:
add comment: the spot cluster is taken down

Reply:
Done. Thanks!